sdk-actors/src/main/java/io/dapr/actors/runtime/AbstractActor.java

```diff
@@ -163,7 +163,7 @@ protected ActorStateManager getActorStateManager() {
           this.actorRuntimeContext.getActorTypeInformation().getName(),
           this.id.toString(),
           actorTimer.getName(),
-          this.actorRuntimeContext.getObjectSerializer().serialize(actorTimer));
+          INTERNAL_SERIALIZER.serialize(actorTimer));
```

Comment:
That's pointing to io.dapr.client.ObjectSerializer.  Don't you want io.dapr.runtime.ObjectSerializer?  That is the one with the custom method for Timers.

Outside scope for this PR, but parent and child shouldn't have the same name, it's error prone.

Reply:
Ok nevermind, it is actually pointing at io.dapr.runtime's version, so it's good.

But like I said, having the name ObjectSerializer for 2 classes is confusing.